### PR TITLE
Fix fitadd and fitdelete command message bugs

### PR DESCRIPTION
**Pull request by jjchee77**

### CODE REVIEW

**Review by dillontkh — APPROVED** [review]
LGTM! Perhaps the usage message could show

```
Parameter
Example
Parameter
Example
```

instead of

```
Parameter
Parameter
Example
Example
```

Would that be less confusing for users?

**Comment by jjchee77** [comment]
> LGTM! Perhaps the usage message could show
> 
> ```
> Parameter
> Example
> Parameter
> Example
> ```
> 
> instead of
> 
> ```
> Parameter
> Parameter
> Example
> Example
> ```
> 
> Would that be less confusing for users?

Good suggestion!

File: src/main/java/seedu/address/logic/messages/FitDeleteCommandMessages.java [hunk 1]
```diff
@@ -13,11 +13,13 @@ public class FitDeleteCommandMessages extends Messages {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes exercise(s) from the client identified "
```
Comment (RyanNgWH) [inline src/main/java/seedu/address/logic/messages/FitDeleteCommandMessages.java:13]: Should not allow multiple exercises to be deleted. However, is fixed in #368

**Review by RyanNgWH — APPROVED** [review]
LGTM!

**Review by ruijietay — APPROVED** [review]
LGTM

**Pull request merged**